app/models/govbox_set.rb

```diff
@@ -104,22 +104,17 @@ def init_news_items
 
   def init_jobs
     if @affiliate.jobs_enabled?
-      job_results = Jobs.search(build_jobs_search_options)&.search_result&.search_result_items
+      job_results = Jobs.search(Keyword:        @query,
+                                Organization:   @affiliate.agency&.joined_organization_codes,
+                                LocationName:   @geoip_info&.location_name,
+                                ResultsPerPage: 10)&.search_result&.search_result_items
       if job_results.present?
         @jobs = JobResultsPostProcessor.new(results: job_results)&.post_processed_results
         @modules << 'JOBS'
       end
     end
   end
 
-  def build_jobs_search_options
-    jobs_options = { Keyword: @query, ResultsPerPage: 10 }
-    org_hash = { Organization: @affiliate.agency&.joined_organization_codes }
-    jobs_options.merge!(org_hash)
-    jobs_options.merge!(LocationName: @geoip_info&.location_name)
-    jobs_options.compact
-  end
-
```

Comment:
build_job_search_options was initially determine the correct value for fields for example the geoip_info based on longitude and latitude.  Now that we are using usajobs this method is not really needed. So I removed it and passed the options directly.